automatic/apachelounge-httpd/tools/helpers.ps1

```diff
@@ -97,9 +102,12 @@ function Install-ApacheService {
 
     $apachePaths = Get-ApachePaths $arguments.destination
 
+    Write-Warning "This command is expected to write debugging output. Chocolately treats that as an error. Please read the error text to confirm whether service installation succeed."
     & $apachePaths.BinPath -k install -n "$($arguments.serviceName)"
```

Comment:
Would something like this fix this up? Given that it does raise things that need fixing, I'd almost be inclined to suggest outputting it as a warning if there was anything found, but that's more processing...

```suggestion
    & $apachePaths.BinPath -k install -n "$($arguments.serviceName)" 2>&1
```

I believe that should work back to 2.0.

Reply:
Example output from a quick test:
![image](https://github.com/chocolatey-community/chocolatey-packages/assets/1975761/6283ed44-2bcc-4a7c-904c-093b6d73b971)

Reply:
Example of highlighting the errors:

```suggestion
    # This command outputs to the error stream. 2>&1 redirects that to output.
    & $apachePaths.BinPath -k install -n "$($arguments.serviceName)" 2>&1 | ForEach-Object {
        if ($_ -match "^\w+\d+: ") {
            Write-Warning "$_"
        }
        else {
            Write-Host "$_"
        }
    }
```

![image](https://github.com/chocolatey-community/chocolatey-packages/assets/1975761/49a468c3-6131-4672-852e-f6c3d493b317)

Reply:
@AdmiringWorm Any thoughts on my pushing the changes suggested in the last comment? I think it makes sense, then we can merge this in.

Reply:
Let us not go with any custom parsing inside the package, so I would say just go with the first option. We can always make changes at a later date if needed.

Reply:
Ah, that's a shame, as it seemed useful - but fair enough, it's a line. I'll pop that in.